src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/ApplicationSummary.vue

```diff
@@ -1,33 +1,142 @@
 <template>
   <v-container>
-    <v-breadcrumbs :items="items" color="primary">
+    <v-breadcrumbs class="breadcrumb" :items="items" color="primary">
       <template #divider>/</template>
     </v-breadcrumbs>
+
     <ApplicationCertificationTypeHeader :certification-types="applicationStore.applications?.[0]?.certificationTypes || []" class="pb-5" />
     <h3>Status</h3>
     <div class="pb-3">It's a 3-step process to apply</div>
+    <!-- Step 1 Start-->
+    <v-card elevation="0" color="#f8f8f8" class="border-top mt-5" rounded="0">
+      <v-card-text>
+        <div class="d-flex" :class="[smAndUp ? 'space-between align-center' : 'flex-column']">
+          <v-row no-gutters>
+            <v-col cols="12"><strong>Step 1</strong></v-col>
+            <v-col cols="12"><strong>Submit Application</strong></v-col>
+          </v-row>
+          <div>
+            <v-icon icon="mdi-check" size="x-large" />
+            Complete
+          </div>
+        </div>
+      </v-card-text>
+    </v-card>
+    <v-card elevation="0" rounded="0" class="border-t border-b">
+      <v-card-text>
+        <div>Completed on {{ formatDate(applicationStatus?.submittedOn as string, "LLL d, yyyy") }}</div>
+      </v-card-text>
+    </v-card>
+    <!-- Step 1 End-->
+    <!-- Step 2 Start-->
+    <v-card
+      elevation="0"
+      :color="step2Progress === IN_PROGRESS ? '#003366' : '#f8f8f8'"
```

Comment:
Any way to reference 'primary' colour here? Also, is #f8f8f8 new? You should be able add as a custom colour (annoyingly in two places) `_colors.scss` and `ecer-theme.ts` if you can reference here

Reply:
hmm seems like it's a new color from the figma requirements. It's called "white-smoke". It's lighter than our lightest-grey color. I've created a constant for this color.